app/assets/javascripts/media_player_wrapper/mejs4_helper_markers.es6

```diff
@@ -304,13 +289,13 @@ class MEJSMarkersHelper {
       .then(response => {
         // Insert the fresh HTML table
         $('#markers').replaceWith(response);
-        // Add event listeners to newly created row
-        t.addMarkersTableListeners();
         // hide marker sections if no markers (first row is header)
         if ($('#markers').find('tr').length === 1) {
           $('#markers_section').collapse('hide');
           $('#markers_heading').hide();
         } else {
+          // Add event listeners to newly created row
```

Comment:
This is an old comment hopefully, "to newly created row"?   It should be applying to all rows.